english/bounce/bounce.md

```diff
@@ -4,7 +4,6 @@ BOUNCE!
 
 straight across the room to bounce off the far marshmallow wall. You continue to bounce from
 wall, to floor, to ceiling, and back again. Finally, before you become sick from all the 
-bouncing around you bounce right through one of the marshmallow walls to find yourself right
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.